pylua/server3.py

```diff
@@ -0,0 +1,14 @@
+import asyncio
+import websockets
+
+async def server_task(websocket, path):
+	message = await websocket.recv()
+	print("=> {}".format(message))
+
+	response = message
+	websocket.close()
+
+start_server = websockets.serve(server_task, 'localhost', 8087)
+
+asyncio.get_event_loop().run_until_complete(start_server)
```

Comment:
@oherasym,  these two calls seems redundant, as far as i understand you shaul use one of them